src/main/webapp/WEB-INF/templates/fragments/head.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+
+<head th:fragment="head">
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="static/js/collector.js" type="text/javascript"></script>
+    <script type="text/javascript">
+        var $SCRIPT_ROOT = "http://localhost:8080/SecureDataCollector-1.0-SNAPSHOT";
+    </script>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link href="static/css/style.css" rel="stylesheet" type="text/css">
+
+    <title>SONET SLOVAKIA</title>
+</head>
+
+<body>
```

Comment:
Skvely nápad dať zdieľaný head súbor, ale má tam byť aj to body, či to nevadí?
Len potom pozor, že ten kód, ktorý je v "/" nesmie byť takto v spoločnom head súbore lebo tam th nejde, a ak je tak nejdú iné stránky